### PR TITLE
Switch to using proper staticfiles_storage.url() call to reference the TinyMCE JS

### PR DESCRIPTION
**Pull request by jameyg**
To fix issue https://github.com/jazzband/django-tinymce/issues/420

### CODE REVIEW

**Review by merwok — APPROVED** [review]

File: tinymce/settings.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import os
```
Comment (claudep) [inline tinymce/settings.py:1]: I think that import can be removed.

**Review by claudep — CHANGES_REQUESTED** [review]
Thanks! You can also include the "Fixes #420" either in the commit message or in the second line of the commit so that the issue will be automatically closed.

**Comment by jameyg** [comment]
@claudep I've amended the commit message to included "Fixes #420" and removed the redundant `import os` call :+1:

**Review by claudep — APPROVED** [review]

**Comment by claudep** [comment]
Thanks!

**Pull request merged**